Packs/CommonScripts/Scripts/PrintRaw/Pipfile

```diff
@@ -0,0 +1,35 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+pytest = "==5.0.1"
+pytest-mock = "*"
+requests-mock = "*"
+pytest-asyncio = "*"
+pytest-xdist = "*"
+freezegun = "*"
+pytest-datadir-ng = "*"
+flake8 = "*"
+
+[packages]
+certifi = "==2019.11.28"
+chardet = "==3.0.4"
+dateparser = "==0.7.4"
+funcy = "==1.14"
+idna = "==2.9"
+olefile = "==0.46"
+python-dateutil = "==2.8.1"
+pytz = "==2019.3"
+regex = "==2020.2.20"
+requests = "==2.23.0"
+six = "==1.14.0"
+tzlocal = "==2.0.0"
+urllib3 = "==1.25.8"
+PySocks = "==1.7.1"
+PyYAML = "==5.3"
```

Comment:
They aren't needed; they're all added by 'demisto-sdk split-yml'.  But why is it a concern now, after so many PR's where it wasn't?  The contribution guide doesn't mention anything about Pipfile needing to be pruned.

Reply:
You are 100% right that this was used to be like this so far, seems like we have an issue there, will let you know if this might cause problems, in the meanwhile keep it as is.